app/views/compute_resources/show/_ovirt.html.erb

```diff
@@ -14,3 +14,13 @@
   <td><%= _("Default VNC Keyboard") %></td>
   <td><%= @compute_resource.keyboard_layout %></td>
 </tr>
+<tr>
+  <td colspan="2">
+    <div class='alert alert-warning'>
+      <%= _('oVirt integration is deprecated.') %>
```

Comment:
I found this post in Ovirt blog: https://blogs.ovirt.org/2022/02/future-of-ovirt-february-2022/

> Last year, Red Hat published an update to the RHV product life cycle, announcing its end of life in August 2026, with  OpenShift Virtualization as its successor. But as a commercial software product, RHV has a different prioritization from oVirt, which is and will always be a community project. To be clear, **while there will be an End of Life for RHV, there is no planned End of Life for oVirt.**

therefore I guess we should display the warning banner only for RHV compute resource.
can we also describe more, e.g: 
```on August 31, 2022, Red Hat Virtualization entered the Maintenance Support Phase. The Maintenance Support Phase runs until August 31, 2024, followed by the Extended Life Phase, which runs until August 31, 2026,``` [Read more](https://access.redhat.com/announcements/6960518)